admin/src/js/controllers/users.js

```diff
@@ -52,6 +52,14 @@ angular.module('controllers').controller('UsersCtrl',
       });
     };
 
+    $scope.showAddMultipleUsersModal = function() {
+      Modal({
+        templateUrl: 'templates/multiple_user_modal.html',
+        controller: 'MultipleUserCtrl',
+        model: {},
+      });
+    };
```

Comment:
Prefer the arrow function syntax when possible.

```suggestion
    $scope.showAddMultipleUsersModal = () => {
      Modal({
        templateUrl: 'templates/multiple_user_modal.html',
        controller: 'MultipleUserCtrl',
        model: {},
      });
    };
```